.eslintrc.js

```diff
@@ -29,6 +29,31 @@ module.exports = {
     "import/resolver": {
       node: {
         paths: ["src"]
+      },
+      extends: ["plugin:jest/recommended", "airbnb"],
+      globals: {
+        Atomics: "readonly",
+        SharedArrayBuffer: "readonly"
+      },
+      parserOptions: {
+        ecmaFeatures: {
+          jsx: true
+        },
+        ecmaVersion: 2018,
+        sourceType: "module"
+      },
+      plugins: ["react", "jest"],
+      rules: {
+        "react/jsx-filename-extension": [1, { extensions: [".js", ".jsx"] }],
+        "jest/no-disabled-tests": "warn",
+        "jest/no-focused-tests": "error",
+        "jest/no-identical-title": "error",
+        "jest/prefer-to-have-length": "warn",
+        "jest/valid-expect": "error",
+        react: ""
+      },
+      rules: {
```

Comment:
You removed the comments here regarding the multiple `rules` property name. Why is that?

Reply:
@adesege am not really sure which comments you are talking about here.